src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -910,27 +1006,34 @@ LibraryServiceApi.prototype.moveBook = function(request, options, callback) {
  * @param {Object=} options
  *   Optional parameters. You can override the default settings for this call, e.g, timeout,
  *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/global.html#CallOptions} for the details.
- * @param {function(?Error, ?Object, ?string)=} callback
- *   When specified, the results are not streamed but this callback
- *   will be called with the response object representing [ListStringsResponse]{@link ListStringsResponse}.
- *   The third item will be set if the response contains the token for the further results
- *   and can be reused to `pageToken` field in the options in the next request.
- * @returns {Stream|Promise}
- *   An object stream which emits a string on 'data' event.
- *   When the callback is specified or streaming is suppressed through options,
- *   it will return a promise that resolves to the response object. The promise
- *   has a method named "cancel" which cancels the ongoing API call.
+ * @param {function(?Error, ?Object)=} callback
+ *   The function which will be called with the result of the API call.
+ *
+ *   The second parameter to the callback is Array of string.
+ *
+ *   When autoPaginate: false is specified through options, the result is
+ *   an object representing [ListStringsResponse]{@link ListStringsResponse}
+ *   and the third item will be set if the response contains the token for the further
+ *   results and can be reused to `pageToken` field in the options in the next request.
+ * @returns {Promise} - The promise which resolves to the response object.
+ *   The promise has a method named "cancel" which cancels the ongoing API call.
+ *   Or this resolves to an object representing [ListStringsResponse]{@link ListStringsResponse}
+ *   when `autoPaginate: false` is specified through options.
  *
  * @example
  *
  * var api = libraryV1.libraryServiceApi();
  *
  * // Iterate over all elements.
- * api.listStrings({}).on('data', function(element) {
- *     // doThingsWith(element)
+ * api.listStrings({}).then(function(resources) {
+ *     for (var i = 0; i < resources.length; ++i) {
+ *         // doThingsWith(resources[i])
+ *     }
+ * }).catch(function(err) {
+ *     console.error(err);
  * });
  *
- * // Or obtain the paged response through the callback.
+ * // Or obtain the paged response.
  * function callback(err, response, nextPageToken) {
```

Comment:
It's going to be the gRPC response object. In this example, it's `ListBooksResponse` as is described in the document for `callback`. In pubsub's `listTopics`, it's `listTopicsResponse`. So that's different from `resources`.